### ISSUE TITLE
[feature] Publish centrifugo gRPC schema to Buf Schema Registry

### ISSUE THREAD

**Issue by kop** [centrifugal/centrifugo#851]
Nowadays, to start using Centrifugo gRPC API, we have to compile protobufs first.
This is a working solution, but this can be improved.

The [Buf Schema Registry](https://buf.build/product/bsr) provides hosting for Protobuf/gRPC definitions, similar to how package managers work. The giant bonus of BSR is its support for [automatic SDKs generation](https://buf.build/docs/bsr/generated-sdks/overview).

By publishing gRPC schemas to BSR, Centrifugo could provide a number of "include-and-use" gRPC SDKs for Go, JS, Rust, Python and other languages.

**Comment by FZambia** [comment]
Done in v5.4.3 - https://github.com/centrifugal/centrifugo/releases/tag/v5.4.3

Example for Go can be found in description of https://github.com/centrifugal/centrifugo/pull/863

@kop please check it out

**Comment by kop** [comment]
Thank you, @FZambia. I've successfully switched to SDK generated from BSR and everything seems to be running smooth.

One issue I noticed is that I was unable to use the latest version of `protocolbuffers/go` plugin (`v1.34.2`). Had to use `v1.33.0` instead. The Go module BSR generates does not define the minimum Go version in `go.mod`, so Go defaults to `v1.16`. Which does not work because the generated code is using newer syntax available starting from `v1.18`.

However, this might be a problem with my setup. I have quite a Frankenstein project with dependencies on modules that use old gRPC-related libraries. Maybe this is the reason Go dependency resolution is not happy in my case.
